staging/src/k8s.io/kms/apis/v2/api.proto

```diff
@@ -49,8 +49,14 @@ message DecryptRequest {
     string uid = 2;
     // The keyID that was provided to the apiserver during encryption.
     // This represents the KMS KEK that was used to encrypt the data.
+    // keyID must satisfy the following constraints:
+    // 1. The keyID is not empty.
+    // 2. The size of keyID is less than 1 kB.
     string key_id = 3;
     // Additional metadata that was sent by the KMS plugin during encryption.
+    // Annotations must satisfy the following constraints:
+    //  1. Annotation key must be a fully qualified domain name that conforms to the definition in DNS (RFC 1123).
+    //  2. The size of annotations keys + values is less than 32 kB.
```

Comment:
The request comes the API server so we do not need to include this information here.